cruise-control-core/src/main/java/com/linkedin/cruisecontrol/servlet/parameters/CruiseControlParameters.java

```diff
@@ -21,7 +21,7 @@
    * Parse the parameters set in HTTP request to initialize object.
    *
    * @param response HTTP response of Cruise Control. Populated in case of a parameter parse exception.
-   * @return True if there has been a failure to parse parameters, false otherwise. If the object is already initialized,
+   * @return True if there has been a failure to parse parameters, {@code false} otherwise. If the object is already initialized,
```

Comment:
```suggestion
   * @return {@code true} if there has been a failure to parse parameters, {@code false} otherwise. If the object is already initialized,
```